wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/license/CodenvyLicenseManager.java

```diff
@@ -108,4 +119,37 @@ public void delete() throws LicenseException {
             throw new LicenseException(e.getMessage(), e);
         }
     }
+
+    /**
+     * Return true if only Codenvy usage meets the constrains of license properties or free usage properties.
+     **/
+    public boolean isCodenvyUsageLegal() throws ServerException, IOException {
+        long actualUsers = adminUserDao.getAll(30, 0).getTotalItemsCount();   // TODO ndp (codenvy/600) fix it get all users number
```

Comment:
1. Dmytro Nochevnov is more cleaner for other developers
2. codenvy/600 issue says nothing about this TODO

I'd rewrite it as `//TODO Replace it with UserManager#getTotalCount when codenvy->jpa-integration branch will be merged to master`